apps/mark-scan/frontend/src/api.ts

```diff
@@ -112,7 +116,11 @@ export const getStateMachineState = {
   useQuery() {
     const apiClient = useApiClient();
     return useQuery(this.queryKey(), () => apiClient.getPaperHandlerState(), {
-      refetchInterval: STATE_MACHINE_POLLING_INTERVAL_MS,
+      refetchInterval: !isFeatureFlagEnabled(
+        BooleanEnvironmentVariableName.DISABLE_VXMARKSCAN_STATE_MACHINE_POLLING
```

Comment:
This double negative is confusing but fits our existing pattern of development flags being opt-in eg. DISABLE_CARD_READER_CHECK, SKIP_PIN_ENTRY etc

Reply:
You could switch the order of the ternary expression to remove the double negative, if that would help